### ISSUE TITLE
Simplification of expressions with NaN?

### ISSUE THREAD

**Issue by rdbyk** [symengine/symengine#1253]
Hi,

I am using a clone of today and noticed that simple expressions with NaN do not get
simplified as expected. Please have a look at the following examples:

-----------------------------

--> Sym("0/0 + x")
 ans  =

   nan + x
                                                                        (1:Add)
-> Sym("0/0*x")
 ans  =

   nan*x
                                                                        (1:Mul)

--> Sym("(0/0)/x")
 ans  =

   nan/x
                                                                        (1:Mul)

--> Sym("0/0-x")
 ans  =

   nan - x
                                                                        (1:Add)

-----------

I expected all these expressions to be simplified automatically to "nan" !?

Best regards,

Dirk

**Comment by isuruf** [comment]
Yes, NaN support is minimal for now. This can be easily fixed once https://github.com/symengine/symengine/pull/1058 is in, which I think will happen this summer.
